File: WindowsFormsApp2/Classes/MockedData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WindowsFormsApp2.Classes
{
    public class MockedData
    {
        public static List<BoxItem> Boxes()
        {
            return new List<BoxItem>()
            {
                new BoxItem() {Length = 25, Unit = Unit.IM},
                new BoxItem() {Length = 26, Unit = Unit.IM},
                new BoxItem() {Length = 27, Unit = Unit.IM}
            };
        }
    }
}

File: UnitsDemo/Classes/MockedData.vb
Namespace Classes
	Public Class MockedData
		Public Shared Function Boxes() As List(Of Box)
			Return New List(Of Box)() From {
				New Box() With {
					.Length = 25,
					.Unit = Unit.IM
				},
				New Box() With {
					.Length = 26,
					.Unit = Unit.IM
				},
				New Box() With {
					.Length = 27,
					.Unit = Unit.IM
				}
			}
		End Function
	End Class
End Namespace
